llvm/include/llvm/Analysis/TargetTransformInfo.h

```diff
@@ -1810,6 +1810,21 @@ class TargetTransformInfo {
   /// \return The maximum number of function arguments the target supports.
   unsigned getMaxNumArgs() const;
 
+  /// Returns options for jump threading.
+  struct JumpThreadingOptions {
```

Comment:
Missing the parsing code and tests for it (e.g. see other pass parameters in PassBuilder.cpp)